src/main/java/net/shadowmage/ancientwarfare/npc/ai/NpcAIAttackMeleeLongRange.java

```diff
@@ -37,6 +38,9 @@ protected void doAttack(double distanceToEntity) {
 			npc.attackEntityAsMob(getTarget());
 			setAttackDelay(getCoolDown());
 			npc.addExperience(AWNPCStatics.npcXpFromAttack);
+			if ((Math.random() < 0.2 && npc instanceof NpcFaction)) {

```

Comment:
We should play the sounds in attackEntityAsMob rather than AI. that should also allow you to override attackEntityAsMob and call the play sound in addition to calling super.attackEntityAsMob and there won't be a need for instanceof check in that case